SU-SWS/decanter
# D8CORE-2544: added (link is external) to the brandbar

# READY FOR REVIEW

# Summary
Adds  `<span class="su-brand-bar__link--external">(link is external)</span>` to the external link in the brand bar
This will announce to a screen reader that this is an external link.

# Needed By (Date)
- Soon

# Urgency
- Not urgent

# Steps to Test

1. Check out this branch in Stanford Basic
2. run npm install in Stanford Basic
3. Run npm run publish to populate /decanter in the Stanford Basic templates directory
4. Inspect the brand bar to verify `<span class="su-brand-bar__link--external">(link is external)</span>`accompanies the external link in the global header.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/D8CORE-2544
- Related PRs: https://github.com/SU-SWS/decanter/pull/716
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@JBCSU 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
